content/docs/develop/cli/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: keptn CLI
+description: Explains how to download and install the keptn CLI as well as which commands are available.
+weight: 30
+icon: help
+keywords: [cli, setup]
+---
+
+In this section, the functionality and commands of the Keptn CLI are described. The Keptn CLI allows installing, configuring, and
```

Comment:
In this paragraph: 

When talking about the `keptn cli`, let`s use a lower case `k`.